src/platforms/ctxlink/platform.c

```diff
@@ -156,6 +156,20 @@ int platform_hwversion(void)
 	return 0;
 }
 
+static char parameters[128] = {0};
+
+const char *platform_wifi_state(int argc, const char **argv)
+{
+	(void)argv;
```

Comment:
This parameter is used in the `if` block below, so please drop this line.

Reply:
Fixed.